# Define the goal
uint32 trocar_id # which trocar to place the tool into
---
# Define the result
uint32 result
---
# Define a feedback message
float32 percent_complete